src/docs/sphinx/developerGuide/Contributing/UsingDocker.rst

```diff
@@ -52,7 +52,7 @@ I like to do
 
     docker run --cap-add=SYS_PTRACE -d --name ${REMOTE_DEV_IMG}-${VERSION} -p 64000:22 -p 64010-64020:64010-64020 ${REMOTE_DEV_IMG}:${VERSION}
 
-that creates the container ``remote-dev-ubuntu18.04-gcc8-156-642``, running instance of ``remote-dev-ubuntu18.04-gcc8:156-642``.
+that creates the container ``remote-dev-ubuntu20.04-gcc9-212-910``, running instance of ``remote-dev-ubuntu20.04-gcc9:212-910``.
 
 - Note that you'll have to access your remote development instance though port ``64000`` (forwarded to standard port ``22`` by docker).
 - Additional ports from ``64010`` to ``64020`` will be open if you need them (multiple instances of ``gdbserver``, remote ``paraview`` connection, ...). 
```

Comment:
should we add some how to example or links to places where it is explained ?

Reply:
We already link to the doc where we explain the TPL tag. What do you have in mind?